test

```diff
@@ -13,6 +13,7 @@ source ./build
 
 TESTABLE="plugins/ipam/dhcp plugins/main/loopback pkg/invoke pkg/ns pkg/skel"
 FORMATTABLE="$TESTABLE libcni pkg/ip pkg/ns pkg/types pkg/ipam plugins/ipam/host-local plugins/main/bridge plugins/meta/flannel plugins/meta/tuning"
+LINT="./..."
```

Comment:
Actually, this won't work because golint needs files or packages or `...` to.

```
Checking golint (informational only)...
read libcni: is a directory
read plugins/ipam/dhcp: is a directory
read plugins/ipam/host-local: is a directory
read plugins/ipam/host-local/backend/allocator: is a directory
read plugins/main/loopback: is a directory
read pkg/invoke: is a directory
read pkg/ns: is a directory
read pkg/skel: is a directory
read pkg/types: is a directory
read pkg/types/current: is a directory
read pkg/types/020: is a directory
read pkg/utils: is a directory
read plugins/main/ipvlan: is a directory
read plugins/main/macvlan: is a directory
read plugins/main/bridge: is a directory
read plugins/main/ptp: is a directory
read plugins/test/noop: is a directory
read pkg/utils/hwaddr: is a directory
read pkg/ip: is a directory
read pkg/version: is a directory
read pkg/version/testhelpers: is a directory
read plugins/meta/flannel: is a directory
read pkg/ipam: is a directory
read pkg/testutils: is a directory
read plugins/meta/flannel: is a directory
```